velox/docs/functions/spark/math.rst

```diff
@@ -0,0 +1,57 @@
+====================================
+Mathematical Functions
+====================================
+
+.. spark:function:: abs(x) -> [same as x]
+
+    Returns the absolute value of ``x``.
+
+.. spark:function:: add(x, y) -> [same as x]
+
+    Returns the result of adding x to y. The types of x and y must be the same.
+    For integral types, overflow results in an error. Corresponds to sparks's operator ``+``.
+
+.. spark:function:: ceil(x) -> [same as x]
+
+    Returns ``x`` rounded up to the nearest integer.  
+    In spark only long, double, and decimal have ceil.
+
+.. spark:function:: divide(x, y) -> [same as x]
+
+    Returns the results of dividing x by y. The types of x and y must be the same.
+    It always performs floating point division. Corresponds to spark's operator ``/``.
+
+.. spark:function:: exp(x) -> double
+
+    Returns Euler's number raised to the power of ``x``.
+
+.. spark:function:: floor(x) -> [same as x]
+
+    Returns ``x`` rounded down to the nearest integer.
+    In spark only long, double, and decimal have floor.
+
+.. spark:function:: multiply(x, y) -> [same as x]
+
+    Returns the result of multiplying x by y. The types of x and y must be the same.
+    For integral types, overflow results in an error. Corresponds to spark's operator ``*``.
+
+.. spark:function:: pmod(n, m) -> [same as n]
+
+    Returns the positive value of ``n`` divided by ``m``.
+
+.. spark:function:: power(x, p) -> double
+
+    Returns ``x`` raised to the power of ``p``.
+
+.. spark:function:: remainder(n, m) -> [same as n]
+
+    Returns the modulus (remainder) of ``n`` divided by ``m``. Corresponds to spark's operator ``%``.
+
+.. spark:function:: round(x, d) -> [same as x]
+
+    Returns ``x`` rounded to ``d`` decimal places using HALF_UP rounding mode.
```

Comment:
> HALF_UP

Perhaps, explain what HALP_UP is. 

`In HALF_UP rounding, the digit 5 is rounded up.`